src/constant/MenuItems.ts

```diff
@@ -0,0 +1,99 @@
+import Adjustment from '@assets/icons/adjustments.svg?react';
+import ChartPie from '@assets/icons/chart_pie.svg?react';
+import ClipboardList from '@assets/icons/clipboard_list.svg?react';
+import Cog from '@assets/icons/cog.svg?react';
+import Collection from '@assets/icons/collection.svg?react';
+import Document from '@assets/icons/document_report.svg?react';
+import Globe from '@assets/icons/globe.svg?react';
+import InboxIn from '@assets/icons/inbox_in.svg?react';
+import LockClosed from '@assets/icons/lock_closed.svg?react';
+import Bag from '@assets/icons/shopping_bag.svg?react';
+import Support from '@assets/icons/support.svg?react';
+import { MenuIcon, MenuItem } from '@types';
+
+export const menuData1: MenuItem[] = [
+    {
+        id: '1',
+        title: 'Overview',
+        icon: ChartPie,
+        to: '/overview',
+    },
+    {
+        id: '2',
+        title: 'Pages',
+        icon: Document,
+        children: [
+            { id: '2-1', title: 'Product List', to: '/product-list' },
```

Comment:
```suggestion
            { id: '21', title: 'Product List', to: '/product-list' },
```

We can add to URLS in ROUTE_PATH constants